docs/src/main/asciidoc/reactive-sql-clients.adoc

```diff
@@ -174,20 +176,19 @@ TIP: You may override the default value of the `myapp.schema.create` property in
 
 Almost ready!
 To initialize the DB in development mode, we will use the client simple `query` method.
-It returns a `CompletionStage` and thus can be composed to execute queries sequentially:
+It returns an `Uni` and thus can be composed to execute queries sequentially:
 
 [source,java]
 ----
-        client.query("DROP TABLE IF EXISTS fruits")
-                .thenCompose(r -> client.query("CREATE TABLE fruits (id SERIAL PRIMARY KEY, name TEXT NOT NULL)"))
-                .thenCompose(r -> client.query("INSERT INTO fruits (name) VALUES ('Orange')"))
-                .thenCompose(r -> client.query("INSERT INTO fruits (name) VALUES ('Pear')"))
-                .thenCompose(r -> client.query("INSERT INTO fruits (name) VALUES ('Apple')"))
-                .toCompletableFuture()
-                .join();
+ client.query("DROP TABLE IF EXISTS fruits")
+    .flatMap(r -> client.query("CREATE TABLE fruits (id SERIAL PRIMARY KEY, name TEXT NOT NULL)"))
```

Comment:
`flatMap` is fine with me, so is the Mutiny flavor with `onItem`. Your call.

My only concern is that the test and the doc are consistent, so all good. :+1:

Reply:
I updated the commit message to explain why I'm ok with `flatMap` and `map` here. Reactive programmers are used to them.